onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -291,9 +293,12 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             if einsum_node is not None:
                 unique_index = einsum_node.input[0]
                 new_edge = "edge_modified_" + unique_index
-                shape_tensor = self.model.convert_list_to_tensor(
```

Comment:
we can remove OnnxModel.convert_list_to_tensor since it is the only place reference it.

Reply:
ok